addons/objectstore.md

```diff
@@ -0,0 +1,121 @@
+---
+description: >
+  Objectstore is a storage solution based on the AWS S3 api and is used for persistent storage. 
+---
+
+# Objectstore / S3
+
+objectstore on NAIS stores files in buckets. On-prem uses rook ceph storage nodes as the storage solution. 
+This is persistent storage and is available in all clusters. The cluster is currently running rook 1.2.x and ceph 14.2.x.
+
+The objectstore / S3 bucket access is secured with access keys and secret keys stored in kubernetes.
+
+## How to
+
+### Ceph object storage user
+
+On NAIS on-prem you're required to add a ceph object storage user. This is done through nais-yaml. After applying changes to the ceph-users.yaml and applying this to the cluster the rook operator will create the necessary secrets used to access the S3 bucket.
```

Comment:
Kanskje gjøre det mer tydelig at vi snakker om github-repoet `nais-yaml`, for eks. `navikt/nais-yaml`.